x-pack/plugins/aiops/public/hooks/use_data.ts

```diff
@@ -45,9 +45,6 @@ export const useData = (
   } = useAiopsAppContext();
 
   const [lastRefresh, setLastRefresh] = useState(0);
```

Comment:
I reckon we should get rid of this local state and use `useRefresh` and `useTimeRangeUpdates` hooks instead.

Reply:
Since `lastRefresh` is also passed on to `use_document_count_stats.ts` it will result in quite a rewrite, that's a bit out of scope for this PR, I'd like to focus on just fixing the bug here.